### ISSUE TITLE
AdamWarmup not working in transformers

### ISSUE THREAD

**Issue by acmilannesta** [CyberZHG/keras-bert#167]
Hi,

I tried to wrap AdamWarmup in TFRobertaModel, which use tensoflow 2.0 and tf keras.
Even if I set environment variable
TF_KERAS = 1
I got the following error:
Could not interpret optimizer identifier:', <keras_bert.optimizers.warmup.AdamWarmup object at 0x000002891F6D3E80>.
Could that be because in the 'backend.py', 
```
if TF_KERAS:
    from tensorflow.python import keras
```
Should be changed to?
```
if TF_KERAS:
    from tensorflow import keras
```

**Comment by acmilannesta** [comment]
I wrote a customized callbacks in keras to bypass the issue. Could you help review if that mimic the behavior of AdamWarmup? Thanks!
